client/src/App.tsx

```diff
@@ -23,7 +24,7 @@ function App(): JSX.Element {
                 <Route exact path="/settings/profile/">
                   <Profile />
                 </Route>
-                <Route exact path="/login" component={Login} />
+                <Route exact path="/login" component={SitterDetails} />
```

Comment:
So you are forwarding login to sitterDetails, for test purposes this is okay, but may I know I you included it in the PR?

Reply:
Actually, it was an oversight. I was supposed to remove it after previewing on my local machine.